docs/src/app/components/pages/components/buttons.jsx

```diff
@@ -58,9 +58,9 @@ export default class ButtonPage extends React.Component {
             desc: 'Override the inline hover color of the button\'s root element.',
           },
           {
-            name: 'label or children',
-            type: 'string (label) or HTML/React elements (children)',
-            header: 'required',
+            name: 'label',
+            type: 'string',
+            header: 'optional',
```

Comment:
it is indeed very confusing with two similar ways of doing the same, and i would agree it should be resolved in the library at some point.

In any case though, for now, `label` is *not* required here - you can perfectly fine instantiate the component without it. What is really missing on the documentation side of things is a separate field to specify intended use of children (if any) for each component